https://issues.redhat.com/browse/ACM-13640 (#6927)
open-cluster-management/rhacm-docs: release_notes/whats_new.adoc

[#global-hub-whats-new]
== multicluster global hub 

> * If you want to have your managed hub clusters to have `local-cluster` enabled, import that managed hub cluster in the hosted mode. See link:../global_hub/global_hub_import_hub_hosted.adoc#global-hub-importing-in-hosted-mode[Importing a managed hub cluster in the hosted mode (Technology Preview)]. 

Reviewer: _If you want to have your managed hub clusters to have `local-cluster` enabled_, (this is off a bit; I think to many "have"??

Do you mean:

If you want your managed hub clusters to have _the_ `local-cluster` enabled, (?)

If so, please change that if it is written in the main documentation, as well. 

Since this is a _release note_, it should be more about  what you can do (active) with the new function. Something like:

You can now have the local-cluster enabled on your managed hub clusters by importing your managed hub cluster in hosted mode. (If that is what you mean here, but it seems hosted mode is the new function so maybe....

You can now import a managed hub cluster in hosted mode to enable your local-cluster. (more active, concise, and emphasizes the new function)

These are just some examples for conciseness and clarity.

Reviewer 2: I think `You can now have the local-cluster enabled on your managed hub clusters by importing your managed hub cluster in hosted mode. ` is good to me.

> * If you want to have your managed hub clusters to have `local-cluster` enabled, import that managed hub cluster in the hosted mode. See link:../global_hub/global_hub_import_hub_hosted.adoc#global-hub-importing-in-hosted-mode[Importing a managed hub cluster in the hosted mode (Technology Preview)]. 

Reviewer: ```suggestion
* You can now have the `local-cluster` enabled on your managed hub clusters by importing your managed hub cluster in hosted mode. See link:../global_hub/global_hub_import_hub_hosted.adoc#global-hub-importing-in-hosted-mode[Importing a managed hub cluster in the hosted mode (Technology Preview)]. 
```

Reviewer: approving based on the developer feedback above so I don't hold it up.
